src/include/misc.h

```diff
@@ -44,6 +44,12 @@
 
 #define WT_MINUTE (60)
 
+/* Time (in minutes) the cache is stuck before WT crashes in diagnostic mode.*/
```

Comment:
These should be defined in the file they are used (unless used across different files, which they are not).

Reply:
WT_MINUTE is used elsewhere (i.e. in `/src/conn/conn_tiered.c`)